content/en/docs/instrumentation/js/getting-started/browser.md

```diff
@@ -40,26 +40,68 @@ Copy the following file into an empty directory and call it `index.html`.
 
 To create traces in the browser, you will need `@opentelemetry/sdk-trace-web`, and the instrumentation `@opentelemetry/instrumentation-document-load`:
 
-```shell
+{{< tabs TypeScript JavaScript >}}
+
+{{< tab >}}
+tsc --init
+npm install --save @opentelemetry/api @opentelemetry/sdk-trace-web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone
+{{< /tab >}}
+
+{{< tab >}}
 npm init -y
 npm install --save @opentelemetry/api @opentelemetry/sdk-trace-web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone
-```
+{{< /tab >}}
+
+{{< /tabs >}}
```

Comment:
> Drop the npm init -y, AFAIK, you don't need it for this simple example app.

I introduced that a while back into the nodejs getting started because I had a node_modules in a parent directory and node is then putting everything there instead of the local directory. Although this is expected behaviour by npm, it's confusing, having a basic `package.json` fixes that. The alternative is `echo {} > package.json`.

Reply:
I will just keep the `npm init -y` and `tsc --init` commands for now

Reply:
> I introduced that a while back into the nodejs getting started because I had a node_modules in a parent directory and node is then putting everything there instead of the local directory. Although this is expected behaviour by npm, it's confusing, having a basic `package.json` fixes that.

How likely is it that a first-time user will have a node_modules folder in the parent directory? I'd prefer that we drop the `npm init -y` command, but I'm not strongly against it. 

@kaiwalyakoparkar - all of the other points I made are still relevant. I'll point this out again in my next review.